x-pack/plugins/observability/public/pages/slo_edit/helpers/process_slo_form_values.ts

```diff
@@ -5,32 +5,56 @@
  * 2.0.
  */
 
-import type { CreateSLOParams, GetSLOResponse } from '@kbn/slo-schema';
+import omit from 'lodash/omit';
+import type { CreateSLOInput, SLOWithSummaryResponse, UpdateSLOInput } from '@kbn/slo-schema';
 
-export function transformGetSloToCreateSloParams(
-  values: GetSLOResponse | undefined
-): CreateSLOParams | undefined {
+import { toDuration } from '../../../utils/slo/duration';
+
+export function transformSloResponseToCreateSloInput(
+  values: SLOWithSummaryResponse | undefined
+): CreateSLOInput | undefined {
   if (!values) return undefined;
 
   return {
-    ...values,
+    ...omit(values, ['id', 'revision', 'createdAt', 'updatedAt', 'summary']),
```

Comment:
💬 These fields are not used by the form and must not be present in the payload for the create or update API.

Reply:
@maryam-saeidi 
> I am wondering if it make sense to omit these fields there as well 🤔

omit these fields where?

Reply:
On the BE side.

Maybe I am missing something but I thought for fields like `'createdAt', 'updatedAt'` we always want to set them on the BE side, so I thought maybe we can ignore these fields even if FE sends them to make sure it will not create an issue.